### DISCUSSION TITLE
Rstudio / gt / targeting the correct header for removal

### DISCUSSION THREAD

**Discussion by cuentodeadas** [rstudio/gt#1926] [Q&A] [answered]
Hi. I am unfortunately restricted from sharing my data but I'm hoping for some help with removing a part of the table that I thought was called the header. I want to remove the part of the table that is targeted in modify_header. The default label for this part of the table would read "Characteristic" and "N = 245". If modified so that labels are empty (label = "") the border for this section still remains. I want to remove it entirely. I would also like to know the correct name of this part of the table (stubhead label? column label?). None of the related rm_ or remove_ commands seem to work. Please advise. Below is my code and table visual to illustrate. 

Thanks very much!

```
WS_subset_data %>%
  tbl_summary(
    include = c(x112_Got_HealthSafety_Training, x1Dummy_var),
    value = c(
      x112_Got_HealthSafety_Training ~ "No",
      x1Dummy_var ~ "Total"),
    label = list(
      x112_Got_HealthSafety_Training = "No" ,
      x1Dummy_var = "Total"),
    statistic = list(
      x112_Got_HealthSafety_Training ~ "{n} ({p}%)",
      x1Dummy_var ~ "{n}")
  ) %>%
  remove_row_type(type = "header") %>%
  modify_header(
    label = "How to remove this?",
    all_stat_cols() ~ "and this?"
  ) %>%
  as_gt() %>%
  tab_header(
    title = "Health and Safety Training",
    subtitle = html("Survey question: When you started working at your current  <br> job site, were you given training about health and safety <br> hazards in your workplace and how to protect yourself?")) %>%
  rm_footnotes() %>%
  gt_theme_espn()
```
![image](https://github.com/user-attachments/assets/2d726bf7-1c9b-4f4e-8266-1ecea084b972)

@rich-iannone

**Comment by olivroy** [comment, marked as answer]
You can use `tab_options(column_labels.hidden = TRUE)`

**Reply by cuentodeadas** [reply]
Thanks!
